src/protocols/secure_channel/CASEServer.cpp

```diff
@@ -152,7 +152,7 @@ void CASEServer::PrepareForSessionEstablishment(const ScopedNodeId & previouslyE
 
 void CASEServer::OnSessionEstablishmentError(CHIP_ERROR err)
 {
-    ChipLogError(Inet, "CASE Session establishment failed: %" CHIP_ERROR_FORMAT, err.Format());
+    ChipLogError(Inet, "CASE Session establishment failed: asdf qwerty %" CHIP_ERROR_FORMAT, err.Format());
```

Comment:
Please undo this change?

Reply:
Yeah that's just cruft from when I was testing, I uploaded this PR without a final pass to remove cruft, so it could be "backed up" and "pre-reviewed."